Dockerfile

```diff
@@ -1,27 +1,47 @@
-FROM python:3.8-slim
-
-RUN apt-get update && apt-get -y install git gcc
-
-RUN useradd -r app_user
-RUN mkdir /app
-RUN chown app_user /app/
-WORKDIR /app
+FROM python:3.10-alpine
```

Comment:
Is the 3.10 / Pipfile.lock upgrade something related to Alpine / CVE hunting or just some maintenance?

Anyway this is good, I'll take benefits of this to extract this change from the public transports branch :smile:

Reply:
I am pretty sure that the same pipfile.lock is usable both in debian and Alpine